### PR TITLE
pydocstyle for log.py

### PR DESCRIPTION
**Pull request by meahow**
This is for #742
I changed the format of the docstrings for cookiecutter/log.py to follow pep257 style

### CODE REVIEW

File: cookiecutter/log.py [hunk 1]
```diff
@@ -18,6 +20,11 @@
 
 
 def configure_logger(stream_level='DEBUG', debug_file=None):
+    """Configure logging for cookiecutter.
+
+    Set up logging to stdout with given level and with DEBUG level for loggin
```
Comment (hackebrot) [inline cookiecutter/log.py]: Typo ``logging``

File: cookiecutter/log.py [hunk 2]
```diff
@@ -18,6 +20,11 @@
...
+    to file if debug_file given
```
Comment (hackebrot) [inline cookiecutter/log.py]: Missing ``.``

**Comment by meahow** [comment]
@hackebrot 
Comments addressed.
Thanks for quick feedback and sorry for being so sloppy :)

File: cookiecutter/log.py [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+"""Module for up logging set up."""
```
Comment (hackebrot) [inline cookiecutter/log.py]: The grammar isn't correct. What are you trying to say?

``Module for setting up logging.``

**Comment by hackebrot** [comment]
Thank you for your contribution @meahow! 🙇

**Pull request merged**